cli/api_utility_commands.go

```
@@ -0,0 +1,52 @@
+	},
+}
+
+var dashboardTokenRotateCommand = &cli.Command{
+	Name:      "rotate",
+	Usage:     "Rotate your claim token (you can do it once a minute)",
+	ArgsUsage: "",
+	Action: func(ctx context.Context, cmd *cli.Command) error {
+		resp := callDashboardApi[UserResponse]("POST", "/web/settings/rotate-claim-token")
+		if resp.Type == "success" {
+			printSuccess("User request was successful")
+			printInfo(resp.Payload.Settings.ClaimToken) // TODO fix
```

golangci-lint (1.62.2)

printf: non-constant format string in call to github.com/husarnet/husarnet/cli/v2.printInfo

(govet)